spec/support/features/page_object/topics.rb

```diff
@@ -149,7 +149,7 @@ def someone_updates_topic
 
     def search_for(title)
       fill_in 'Search', with: title
-      find('.search-bar input[type="submit"]').click
+      find('.user-navigation--search input[type="submit"]').click
```

Comment:
:hand: 


![](https://www.evernote.com/shard/s330/sh/62b9acc1-2ff0-4727-82b6-851a9d6f35ed/cc6585d24cf22ce2c53a69dda13647f6/deep/0/Main-Window-and-Refactor-CSS---Design-header-by-kylefiedler---Pull-Request--77---jayroh-thredded.png)